scripts/ci-release.mjs

```diff
@@ -140,12 +140,10 @@ async function prodMode() {
   }
 
   try {
-    await execa('npx', ['lerna', 'publish', `--otp=${totpCode}`, '--yes'], {
-      stdin: process.stdin,
-      env: {
-        GH_TOKEN: githubToken
-      }
-    })
+    await execa('npx',
+      ['lerna', 'publish', `--otp=${totpCode}`, '--no-push', '--no-git-tag-version', '--no-changelog', '--yes'],
```

Comment:
`lerna version` is the creating the files/tags

Reply:
https://github.com/lerna/lerna/blob/v4.0.0/commands/version/README.md#--no-git-tag-version
https://github.com/lerna/lerna/blob/v4.0.0/commands/version/README.md#--no-changelog

Reply:
```suggestion
      ['lerna', 'publish', 'from-package', `--otp=${totpCode}`, '--no-push', '--no-git-tag-version', '--no-changelog', '--yes'],
```